lib/data/repositories/discover_repository.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:farmsmart_flutter/data/managers/firestore_manager.dart';
+import 'package:farmsmart_flutter/data/model/article_entity.dart';
+
+class DiscoverRepository {
```

Comment:
Repository should reference to what data provides...
in this case, it provides Articles... so it should be an ArticlesRepository... 
I suggest that renaming.

Reply:
Done!